web/rainmaker/dev-packages/egov-bpa-dev/src/ui-config/screens/specs/oc-bpa/search-preview.js

```diff
@@ -310,7 +310,7 @@ const setDownloadMenu = async (action, state, dispatch, applicationNumber, tenan
 const getRequiredMdmsDetails = async (state, dispatch) => {
   let mdmsBody = {
     MdmsCriteria: {
-      tenantId: getTenantId(),
+      tenantId: getTenantId().split('.')[0],
```

Comment:
use commonConfig.tenantId if you want to get state tenant
import commonConfig from "config/common.js";